cmd/ndm_daemonset/probe/sysfsprobe.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2020 OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+
+Find queue sysfs doc here.
+https://www.kernel.org/doc/Documentation/block/queue-sysfs.txt
+
+*/
+
+package probe
+
+import (
+	"io/ioutil"
+	"strconv"
+	"strings"
+
+	"github.com/openebs/node-disk-manager/blockdevice"
+	"github.com/openebs/node-disk-manager/cmd/ndm_daemonset/controller"
+	"github.com/openebs/node-disk-manager/pkg/util"
+	"k8s.io/klog"
+)
+
+const (
+	sysfsProbePriority = 4
```

Comment:
Can we change the priority. Currently, the priority is `seachest > smart > capacity`. Since seachest is disabled by default in most installations, I think the new priority should be `sysfs > smart > seachest`

Reply:
done